technologies/convince/README.md

```diff
@@ -3,14 +3,246 @@
+### Introduction 1
 
-In the folder you will find the following files:
+This is a simple example consisting of a battery that is drained at a fixed rate and the logic to charge it when it is low.
 
-- `main.xml` - The main file referencing the other files and defining some global system parameters.
-- __`battery_drainer.scxml`__ - The state machine that models the battery which is drained.
+### System description 1
+
+The folder `technologies/convince/task_1` contains all the necessary files for this example. In the folder you will find the following files:
+
+- [`main.xml`](task_1/main.xml) - The main file referencing the other files and defining some global system parameters.
+- __[`battery_drainer.scxml`](task_1/battery_drainer.scxml)__ - The state machine that models the battery which is drained.
+- __[`battery_manager.scxml`](task_1/battery_manager.scxml)__ - Model of the manager that will evaluate the battery level.
+- __[`battery_properties.jani`](task_1/battery_properties.jani)__ - The file defining the properties defined in [Linear Temporal Logic (LTL)](https://en.wikipedia.org/wiki/Linear_temporal_logic) to be checked by the model checker:
+  - `battery_depleted`- The battery level is eventually <= 0.
+  - `battery_over_depleted` - The battery level is eventually <= -1.
+  - `alarm_on` - The alarm is eventually on.
+
+### Task 1.1: Running the example
+
+To run the example, navigate to the `task_1` folder:
+
+```bash
+cd $ROS_WS/src/technologies/convince/task_1
+```
+
+First, you need to translate all of the files mentioned above into a single JANI model file:
+
+```bash
+as2fm_scxml_to_jani main.xml
+```
+
+Note that there is now additionally a file `main.jani` in the folder.
+
+Then, you can run the model checker on the generated JANI file, specifying the property to check:
+
+```bash
+smc_storm --model main.jani --properties-names battery_depleted
+```
+
+This will check the property `battery_depleted` and output the result.
+We expect the property to hold, if our system is correct.
+This means that in all the traces that the model checker explores, the battery level will eventually reach 0.
+The output should look like this:
+
+```txt
+CONVINCE Statistical Model Checker
+Checking model: main.jani
+Property "battery_depleted": Pmin=? [F ((topic_level_msg.ros_fields__data <= 0) & topic_level_msg.valid)];
+Result: 1
+```
+
+This means that the property holds with a probability of `1`, which is exactly what we expected.
+
+Let's look at the property we get as an output in some more detail:
+
+```txt
+Pmin=? [F ((topic_level_msg.ros_fields__data <= 0) & topic_level_msg.valid)]
+```
+
+Here
+
+- `Pmin=?` means that we are looking for the minimum probability that the property holds.
+- `F` is the temporal operator "finally" in [Linear Temporal Logic (LTL)](https://en.wikipedia.org/wiki/Linear_temporal_logic) , which says the property holds at some point in the future.
```

Comment:
Other people call this operator "eventually", not sure it makes sense to specify this here

Reply:
I was just happy that it matched with wikipedia. ;)